src/testing/convection_diffusion_explicit_periodic.h

```diff
@@ -0,0 +1,38 @@
+#ifndef __CONVECTION_DIFFUSION_EXPLICIT_PERIODIC_H__
+#define __CONVECTION_DIFFUSION_EXPLICIT_PERIODIC_H__
+
+#include "tests.h"
+#include "dg/dg.h"
+#include "parameters/all_parameters.h"
+
+namespace PHiLiP {
+namespace Tests {
+
+template <int dim, int nstate>
+/// Convection Diffusion periodic unsteady test (currently only diffusion)
+class ConvectionDiffusionPeriodic: public TestsBase
+{
+public:
+        /// delete
+        ConvectionDiffusionPeriodic() = delete;
```

Comment:
```suggestion
        /// Destructor
        ~ConvectionDiffusionPeriodic() {};
```

Reply:
also fix indentation throughout this header file

Reply:
fixed